auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -269,15 +276,21 @@ public SignInIntentBuilder createSignInIntentBuilder() {
     public static class IdpConfig implements Parcelable {
         private final String mProviderId;
         private final List<String> mScopes;
+        private final Bundle mParams;
 
-        private IdpConfig(@SupportedProvider @NonNull String providerId, List<String> scopes) {
+        private IdpConfig(
+                @SupportedProvider @NonNull String providerId,
+                List<String> scopes,
+                Bundle params) {
             mProviderId = providerId;
             mScopes = Collections.unmodifiableList(scopes);
+            mParams = params;
         }
 
         private IdpConfig(Parcel in) {
             mProviderId = in.readString();
             mScopes = Collections.unmodifiableList(in.createStringArrayList());
+            mParams = in.readBundle();
```

Comment:
This should be `in.readBundle(getClass().getClassLoader())` to make the build pass. Since we're just passing in strings right now we don't actually need this, but it'll ensure we're future proof (and make lint happy).

Reply:
Ah, just noticed the CONTRIBUTING.md file. Apologies - just beginning contribution to the library and obviously wasn't paying full attention. Made the change and build should be passing now. Thanks for the review by the way!